addons/web/static/src/js/report/report.js

```diff
@@ -13,7 +13,7 @@ $(document.body)
     .addClass('container-fluid')
     .removeClass('container');
 
-var web_base_url = $('html').attr('web-base-url');
+var web_base_url = window.origin;
```

Comment:
I think the point of all this code, python included, is to use the url as a token to ensure that the iframe's code comes from the right server.
Forcing the trusted url to be window.origin seems to be bypassing this mechanism overall. Here, you just ensure that both origin are the same, but not that they are correct or safe

Reply:
I don't see the risk... this diff is enough imo

You just want to check that you know who is talking to you...
If it is the same url... whatever the base url, current website, ... It seems ok

maybe @mart-e from security will have an objection...

Reply:
Sorry can't say, I am not confortable either with the intention from 752cd0a482d1302131003b3066b3031356ddbf30 and the implications of the change...